modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -2861,38 +2927,55 @@ void GDScriptAnalyzer::reduce_identifier_from_base(GDScriptParser::IdentifierNod
 					p_identifier->set_datatype(member.constant->initializer->get_datatype());
 					p_identifier->source = GDScriptParser::IdentifierNode::MEMBER_CONSTANT;
 					p_identifier->constant_source = member.constant;
-					break;
+					return;
```

Comment:
I replaced a bunch of breaks here by returns - all the unit tests pass, but could someone else double-check that this is safe? I feel like it should be. I don't think any extra work is done after this switch.

Reply:
i don't see why? theres a return statement on line 2956
the previous code flow looks better to me because its clear that if the class has the member, it always returns

Reply:
You are totally right! Clearly haven't been sleeping well and absolutely missed it. Thank you!

Reply:
sleep is the most important thing ❤️